spec.md

```diff
@@ -2412,12 +2418,15 @@ Module `Foo` contains a layer block that creates a node computed from a port def
 FIRRTL version 4.0.0
 ;; snippetbegin
 circuit Foo:
-  layer Bar, bind:       ; Declaration of layer Bar with convention "bind"
+  ; Declaration of layer Bar with convention "bind"
+  layer Bar, bind:
 
   public module Foo:
     input a: UInt<1>
 
-    layerblock Bar:      ; Declaration of a layer block associated with layer Bar inside module Foo
+    ; Declaration of a layer block associated with layer Bar inside module
+    ; Foo
```

Comment:
Could you manually wrap at "inside module Foo" instead of at "Foo" to avoid a [typographic orphan](https://en.wikipedia.org/wiki/Widows_and_orphans)?